knotx-server/src/main/resources/io.knotx.KnotxServer.json

```diff
@@ -51,7 +51,7 @@
       "routing": {
         "GET": [
           {
-            "path": "/content/.*",
+            "path": ".*\\.html",
```

Comment:
I am not sure that we should limit default routing to `.*\\.html`. You can have some other URL policy which breaks this rule. Someone can set rewrite rules like this for example: [http://knotx.io/hello-rest-service/](http://knotx.io/hello-rest-service/). In my opinion last routing entry should be `.*`.